applications/vanilla/controllers/class.postcontroller.php

```diff
@@ -540,7 +540,7 @@ public function comment($DiscussionID = '') {
 
                 $Description = val('Description', $PageInfo, '');
                 $Images = val('Images', $PageInfo, []);
-                $LinkText = t('EmbededDiscussionLinkText', 'Read the full story here');
+                $LinkText = t('EmbededDiscussionLinkText', t('Read the full story here'));
```

Comment:
Can't you replace the call to `fetchPageInfo()` above to `DiscussionModel::fetchPageInfo()`. That is what I was asking for in the original issue.

Reply:
I've refactored into using `DiscussionModel->fetchPageInfo()`. I took a look at some communities that embed comments (coming from Wordpress or any other platform) and it seems that the title twice (as discussion name and within the content) it's not as much of an issue as I thought, so I left as it is.

Now both `PostController->comment()` and `DiscussionController->refetchPageInfo()` handle data the same way.